src/install/dracut-install.c

```diff
@@ -1164,6 +1164,10 @@ static int parse_argv(int argc, char *argv[])
                 }
         }
 
+        if (arg_loglevel >= 0) {
+                log_set_max_level(arg_loglevel);
```

Comment:
`arg_loglevel == -1` at this point of execution.

Reply:
Not if `-v --verbose` or `--debug` are specified.

Reply:
Ah, you moved it to `parse_argv()` .... sorry, didn't parse the patch correctly in the first place.